examples/deepspeed/gpt_neox/zero3.yaml

```diff
@@ -9,13 +9,14 @@ hyperparameters:
   search_world_size: false
   conf_dir: /gpt-neox/configs
   conf_file:
-      - medium.yml
+      - 2-7B.yml
```

Comment:
Where is `2-7B.yaml` located? I don't see it in the example files.

Reply:
this config is built in the corresponding docker image.